### DISCUSSION TITLE
Request: Typescript support

### DISCUSSION THREAD

**Discussion by homoky** [FlatIO/api-client-js#149] [Ideas]
It would be great if js client could by type safe with typescript support. Otherwise we can just use the API.

**Comment by gierschv** [comment]
Thanks for your feedback @homoky ! We will probably switch to a TypeScript-supported SDK in the future.

In the meantime you can also generate it with [OpenAPI generator](https://github.com/OpenAPITools/openapi-generator) that includes support for various TS clients:

> Typescript (AngularJS, Angular (2.x - 11.x), Aurelia, Axios, Fetch, Inversify, jQuery, Nestjs, Node, redux-query, Rxjs)

**Comment by homoky** [comment]
Thanks for the info. It works great. We can keep this issue open for another users.
